testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -575,6 +575,20 @@ public void testUnionDistinctViews()
         }
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testHivePartitionViews()
+    {
+        onHive().executeQuery("DROP VIEW IF EXISTS hive_partition_view");
+        onTrino().executeQuery("CREATE TABLE test_hive_partition(some_id VARCHAR(25), ds VARCHAR(25)) WITH (format='TEXTFILE', partitioned_by=array['ds'])");
```

Comment:
nit: Remove unrelated `format` table property. I know `partitioned_by` is also not mandatory, but no need to remove it (I suppose it's the common use case).